guessTheNumber.js

```diff
@@ -0,0 +1,19 @@
+var playerNumber = prompt("I'm thinking of a number between 1 and 100. Can you guess the number I'm thinking of?")
+
+min = 1
+max = 100
+var randomize =  Math.random() * (max-min) + min;
```

Comment:
Consider the use of a noun for variables - `randomize` is a verb, which makes it sound as if this is doing something (which makes me think that this is the name of a function, which does things, rather than a variable).  I suggest `randomNumber`.